engines/director/director.h

```diff
@@ -285,24 +285,25 @@ class DirectorEngine : public ::Engine {
 // An extension of MacPlotData for interfacing with inks and patterns without
 // needing extra surfaces.
 struct DirectorPlotData {
-	DirectorEngine *d;
-	Graphics::ManagedSurface *dst;
+	DirectorEngine *d = nullptr;
+	Graphics::ManagedSurface *dst = nullptr;
 
 	Common::Rect destRect;
 	Common::Point srcPoint;
 
-	Graphics::ManagedSurface *srf;
-	MacShape *ms;
+	Graphics::ManagedSurface *srf = nullptr;
+	MacShape *ms = nullptr;
 
-	SpriteType sprite;
-	InkType ink;
-	uint32 colorWhite;
-	uint32 colorBlack;
-	int alpha;
+	SpriteType sprite = kInactiveSprite;
+	bool oneBitImage = false;
+	InkType ink = kInkTypeCopy;
+	uint32 colorWhite = 255;
```

Comment:
Are you sure we should not take it from the WM:
```C
colorWhite = d->_wm->_colorWhite;
```

Reply:
Fixed, I missed that it was defined in the constructor.